### PR TITLE
Adding Saurav Khare

### PR DESCRIPTION
**Pull request by SauravKhare**
Adding Saurav Khare

### CODE REVIEW

**Review by BlakeCampbells — CHANGES_REQUESTED** [review]
Will you please add a `uses` URL to your website? It should explains what software/hardware you use.

File: src/data.js [hunk 1]
```diff
@@ -1,6 +1,34 @@
 // keep it commonjs export
 // Data is validated against the following schema https://github.com/wesbos/awesome-uses/blob/master/scripts/utils.js#L53-L68
 module.exports = [
+  {
+    name: 'Saurav Khare',
+    description: 'Software Engineer & Frontend Developer',
+    url: 'https://sauravkhare.xyz/uses',
```
Comment (BlakeCampbells) [inline src/data.js:7]: It looks like this is an empty page right now. Will, you see if you can fix it @SauravKhare?

**Review by BlakeCampbells — CHANGES_REQUESTED** [review]

File: src/data.js (hunk 1 above)
Comment (SauravKhare) [reply src/data.js:7]: The issue is fixed.

**Review by BlakeCampbells — APPROVED** [review]

**Comment by BlakeCampbells** [comment]
@SauravKhare Awesome thanks for fixing that!

**Pull request merged**